apps/poppler/.env.docker

```
@@ -1 +1 @@
-POPPLER_VERSION=25.03.0-5+deb13u2
+POPPLER_VERSION=25.03.0-5+deb13u3
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)